cobalt/android/apk/app/src/main/java/dev/cobalt/coat/javabridge/CobaltJavaScriptAndroidObjectExample.java

```diff
@@ -0,0 +1,39 @@
+package dev.cobalt.coat.javabridge;
+
+import static dev.cobalt.util.Log.TAG;
+
+import android.content.Context;
+import android.util.Log;
+
+/**
+ * A simple example of implement CobaltJavaScriptAndroidObject.
+ */
+public class CobaltJavaScriptAndroidObjectExample implements CobaltJavaScriptAndroidObject {
```

Comment:
I would remove the "example" parts, and only present the printIsAmatiDevice()
(maybe call the class AmatiDeviceInspector or some such?) Of course also
rename `example.js`.

Reply:
renamed the java file as well as the js file.